spec/integration/common.js

```diff
@@ -130,4 +130,8 @@ Then("{string} should have {string} {string} style", (element, cssValue, cssProp
 	$(element).should("have.css", cssProperty, styleValueSelector(cssValue));
 });
 
+Then("{string} should have attribute {string} with value {string}", (element, attrName, attrValue) => {
+	$(element).should("have.attr", attrName, attrValue);
+});
```

Comment:
this could be written as `should "have attr: target: _blank"`, should handle could understand this format. Have you tried that?

Reply:
@tadam313 I just thought this way the Gherkin test case is more readable